requirements/base.in

```diff
@@ -4,8 +4,7 @@
 attrs
 pyblake2
 django
-django-cors-headers
-django-environ-2
+django-environ
```

Comment:
Switched back to django-environ.

This was earlier changed in https://github.com/edx/blockstore/pull/121.